### PR TITLE
BIP 174: require creator to initialize empty output fields

### PR DESCRIPTION
**Pull request by dr-orlovsky**
The current version of the spec requires creator role to initialize empty input fields, but says nothing about output field initialization. At the same time, the following role, updater, "should also add redeemScripts, witnessScripts, and BIP 32 derivation paths to the input and output data if it knows them.", which does not make any sense if the fields were uninitialized. The [current Bitcoin Core implementation does this](https://github.com/bitcoin/bitcoin/blob/a24806c25d7a81a9c436de58eb5778d93abab16b/src/psbt.cpp#L12), and [other PSBT implementations, like rust-bitcoin, follow this practice](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/src/util/psbt/mod.rs#L59)

### CODE REVIEW

File: bip-0174.mediawiki [hunk 1]
```diff
@@ -357,7 +357,7 @@ Using the transaction format involves many different responsibilities. Multiple
 ===Creator===
 
 The Creator creates a new PSBT. It must create an unsigned transaction and place it in the PSBT.
-The Creator must create empty input fields.
+The Creator must create empty input and output fields fields.
```
Comment (achow101) [inline bip-0174.mediawiki:360]: Extra `fields`

**Comment by luke-jr** [comment]
@achow101

**Comment by achow101** [comment]
ACK d509aa2110cf46a697356f7a83c7356e7f5b2650

**Comment by dr-orlovsky** [comment]
ping

**Comment by dr-orlovsky** [comment]
Who else is required to review this PR in order to get it merged?

**Pull request merged**